test/utils.py

```diff
@@ -172,8 +172,8 @@ def submit_and_cancel(backend: IBMBackend) -> IBMJob:
     Returns:
         Cancelled job.
     """
-    qobj = bell_in_qobj(backend=backend)
-    job = backend.run(qobj)
+    circuit = transpile(ReferenceCircuits.bell(), backend=backend)
```

Comment:
It's still used [here](https://github.com/Qiskit-Partners/qiskit-ibm/blob/main/test/utils.py#L144) in `submit_job_bad_shots` - unless this test should be refactored somehow as well?

Reply:
Since `submit_job_bad_shots` is just used to cause a validation error, ideally it'd be replaced by something that doesn't require manipulating qobj. But we can do that later.